.github/scripts/check_diff.py

```diff
@@ -137,9 +137,7 @@ def _get_configs_for_single_dir(job: str, dir_: str) -> List[Dict[str, str]]:
         py_versions = ["3.9", "3.13"]
 
     elif dir_ == "libs/community" and job == "extended-tests":
-        # community extended test resolution in 3.12 is slow
-        # even in uv
```

Comment:
@efriis FYI removing this cap because dependency resolution appears to go reasonably fast in 3.12 now.